changes/process_changes.py

```diff
@@ -83,42 +90,42 @@ def header(title, header_char):
         """
         Format a header.
         """
-        return "{}\n{}\n".format(title, header_char * len(title))
+        return '{}\n{}\n'.format(title, header_char * len(title))
 
     def field(name, value):
         """
         Format a field list entry.
         """
-        return ":{}:\n    {}".format(name, value)
+        return ':{}:\n    {}'.format(name, value)
 
     def format_apis(apis):
         """
         Format the API field content. If absent, return "all".
         """
         if apis:
-            return ", ".join(apis)
+            return ', '.join(apis)
         else:
-            return "all"
+            return 'all'
 
-    entries = sorted((e for e in all_entries() if e["type"] == change_type),
+    entries = sorted((e for e in all_entries() if e['type'] == change_type),
                      key=lambda e: e['date'], reverse=True)
 
     if entries == []:
         return
 
-    print(header("Libadalang API {}".format(header_chunk), "#"))
+    print(header('Libadalang API {}'.format(header_chunk), '#'))
 
     for entry in entries:
-        print(header(entry["title"], "="))
-        print(entry["description"])
-        print(field("tn", entry["tn"]))
-        print(field("apis", format_apis(entry.get("apis"))))
+        print(header(entry['title'], '='))
```

Comment:
I'm a bit troubled by your systematic replacement of double quotes by single quotes. Is there a reason/style guide you're following here ? There's nothing in PEP8 about it, nor in our own Python style guide.

In general if you feel strongly enough about a style convention to silently go over other people's code after the fact and modify it, it would be good to communicate about it.

Reply:
As discussed live: this is just for consistency. In the future we should decide on one style and automatically check it (at least on pure Python source files).